src/app/components/ChartbeatAnalytics/utils/index.ts

```diff
@@ -57,9 +58,11 @@ export const getType = (pageType: PageTypes | 'index', shorthand = false) => {
       return 'article-sfv';
     case MEDIA_ASSET_PAGE:
       return 'article-media-asset';
-    case AUDIO_PAGE:
     case MEDIA_PAGE:
+    case AUDIO_PAGE:
       return 'Radio';
+    case TV_PAGE:
+      return 'TV';
```

Comment:
I need to determine if this is correct... I think we need to check the chartbeat data for where the service does not match the producer - perhaps Indonesia?

Reply:
This code will not be invoked for the services which currently support TV, as this is only required when the service does not match the producer

An example of this is indonesian audio, where the producer = INDONESIAN but the service = indonesia

So let's leave this as is for now.